web/firefoxcom.js

```diff
@@ -285,6 +285,47 @@ class MozL10n {
   window.addEventListener("editingaction", handleEvent);
 })();
 
+if (
+  typeof PDFJSDev === "undefined"
+    ? window.isGECKOVIEW
+    : PDFJSDev.test("GECKOVIEW")
+) {
+  (function listenQueryEvents() {
+    window.addEventListener("pdf.js.query", async evt => {
+      const {
+        detail: { queryId },
+      } = evt;
```

Comment:
Nit: I'd probably do the following, since that's a tiny bit easier on the eyes.
```suggestion
    window.addEventListener("pdf.js.query", async ({ detail }) => {
      const { queryId } = detail;
```